packages/lexical-playground/src/plugins/ImagesPlugin.js

```diff
@@ -34,15 +37,15 @@ export default function ImagesPlugin(): React$Node {
 
     return editor.registerCommand(
       INSERT_IMAGE_COMMAND,
-      () => {
+      (payload) => {
         const selection = $getSelection();
         if ($isRangeSelection(selection)) {
           if ($isRootNode(selection.anchor.getNode())) {
             selection.insertParagraph();
           }
           const imageNode = $createImageNode(
-            yellowFlowerImage,
-            'Yellow flower in tilt shift lens',
+            payload?.src || yellowFlowerImage,
+            payload?.altText || 'Yellow flower in tilt shift lens',
```

Comment:
For accessibility reasons, I'd make both of these required type arguments, so we don't potentially add the wrong above alt text to any image URL from Unsplash, if only the `src` is supplied.
```suggestion
            payload.src ,
            payload.altText,
```